CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
+## Release 1.1.2
+
+* **[Bug fix]** Fixed UI controls positioning and scrollbar behavior
```

Comment:
The CALEBE TRUE 1.1.2 are disponibility to release and modifications if necessary.